src/components/Button/Button.js

```diff
@@ -0,0 +1,204 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import ListPropType from '../../prop-types/list';
+import { ButtonIconPosition, ButtonType, ButtonVariant } from './ButtonEnums';
```

Comment:
I like this a lot – I'm going to apply the same concept to the Avatar collection of components